pkg/controller/oneagent/util.go

```diff
@@ -89,6 +89,8 @@ func copyDaemonSetSpecToOneAgentSpec(dsSpec *appsv1.DaemonSetSpec, crSpec *dynat
 	}
 	// PriorityClassName
 	crSpec.PriorityClassName = dsSpec.Template.Spec.PriorityClassName
+	// DNSPolicy
```

Comment:
_Guessing that this goes into master, so refactor changes can be incorporated._

What use are these comments? they are throughout the func. Redundant and can be removed.

Reply:
From what I can see they are used for reference to recognize which attributes from the OneAgent spec are being taken care of when creating the DaemonSet. From my point of view, these are fine.

Reply:
> From what I can see they are used for reference to recognize which attributes from the OneAgent spec are being taken care of when creating the DaemonSet

that should be done while defining the struct; not while setting fields for the same. i still suggest they seem redundant. 
For a special case when field and attribute names do not match, these comments can be added.